aip/general/0193.md

```diff
@@ -62,11 +62,120 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
+### Error messages
 
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+For each error, the service **must** populate the `message` field on
+[`google.rpc.Status`][Status]. This error message,
+
+- is a developer-facing, human-readable "debug message"
+- is presented in the service's native language
+- both explains the error and offers an actionable resolution to it
+  ([citation](https://cloud.google.com/apis/design/errors#error_model))
```

Comment:
nit: it would be nice to consolidate from cloud.google.apis/design/ to here, vs link out. Maybe a future issue?

Reply:
Filed https://github.com/aip-dev/google.aip.dev/issues/1108